apps/mobile/src/modules/Contacts/ContactList.tsx

```diff
@@ -0,0 +1,342 @@
+import {useQueryClient} from '@tanstack/react-query';
+import {addContacts, Contact, getContacts, useEditContacts, useProfile} from 'afk_nostr_sdk';
+import React, {useEffect, useState} from 'react';
+import {
+  Dimensions,
+  Image,
+  Modal,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+
+import {useTheme} from '../../hooks';
+import {useToast} from '../../hooks/modals';
+
+interface ContactListProps {
+  onClose: () => void;
+}
+
+export const ContactList: React.FC<ContactListProps> = ({onClose}) => {
+  const [nostrAddress, setNostrAddress] = useState('');
+  const [activeTab, setActiveTab] = useState('all'); // 'all' or 'add'
+  const [storedContacts, setStoredContacts] = useState<Contact[]>([]);
+  const {theme} = useTheme();
+  const {showToast} = useToast();
+  const editContacts = useEditContacts();
+  const queryClient = useQueryClient();
+
+  // Destructure refetch from useProfile hook
+  const {data: profile, refetch} = useProfile({publicKey: nostrAddress});
+
+  // Fetch contacts when component mounts
+  useEffect(() => {
+    const fetchContacts = () => {
+      const contactsData = getContacts();
+      if (contactsData) {
+        setStoredContacts(JSON.parse(contactsData));
+      }
+    };
+    fetchContacts();
+  }, []);
+
+  const styles = StyleSheet.create({
```

Comment:
Please refacto it, useStyles(stylesheet) with the stylesheet import like the styles.ts